tools/cbmc/proofs/HTTP/IotHttpsClient_ReadHeader/Makefile.json

```diff
@@ -45,9 +40,7 @@
   ],
   "DEF":
   [
-    "MAX_ACCEPTED_SIZE={MAX_ACCEPTED_SIZE}",
-    "CBMC_OBJECTID_BITS={CBMC_OBJECTID_BITS}",
-    "CBMC_MAX_OBJECT_SIZE={CBMC_MAX_OBJECT_SIZE}"
+    "MAX_ACCEPTED_SIZE={MAX_ACCEPTED_SIZE}"
```

Comment:
Where is MAX_ACCEPTED_SIZE defined?  I can't see what the change here is.

Reply:
Line 6 of the same file. The change was just to remove these 2 lines:

```
    "CBMC_OBJECTID_BITS={CBMC_OBJECTID_BITS}",
    "CBMC_MAX_OBJECT_SIZE={CBMC_MAX_OBJECT_SIZE}"
```

These macros now come from `#include ""cbmc.h"`